### DISCUSSION TITLE
Autocomplete component

### DISCUSSION THREAD

**Discussion by lkbrian** [chakra-ui/chakra-ui#9061] [Ideas] [answered]
when I have huge data sets and I want to use selection.
eg.. country selection where there are like 200+ options using Select tag can be quite tiresome given you have multiple instances being used

**Comment by FilipRybinski** [comment]
https://github.com/chakra-ui/chakra-ui/pull/9199

**Comment by segunadebayo** [comment, marked as answer]
Use Combobox. https://chakra-ui.com/docs/components/combobox
